WNPRC_EHR/resources/queries/study/chairing.js

```diff
@@ -88,3 +88,49 @@ function onUpsert(helper, scriptErrors, row, oldRow){
 
 
 }
+function setDescription(row,helper){
+    var description = new Array();
+
+    if (row.description){
+        description.push(EHR.Server.Utils.nullToString(row.description));
+    }
+    if (row.location){
+        console.log(JSON.stringify(row.location));
```

Comment:
not sure if intentional, but there is some logging left in here

Reply:
Yes this was overlooked